cmd/aro/rp.go

```diff
@@ -108,13 +107,11 @@ func rp(ctx context.Context, log, audit *logrus.Entry) error {
 	if err := env.ValidateVars(envDatabaseAccountName); err != nil {
 		return err
 	}
-
 	dbAccountName := os.Getenv(envDatabaseAccountName)
-	clientOptions := &policy.ClientOptions{
-		ClientOptions: _env.Environment().ManagedIdentityCredentialOptions().ClientOptions,
-	}
+
 	logrusEntry := log.WithField("component", "database")
-	dbAuthorizer, err := database.NewMasterKeyAuthorizer(ctx, logrusEntry, msiToken, clientOptions, _env.SubscriptionID(), _env.ResourceGroup(), dbAccountName)
+	scope := []string{fmt.Sprintf("https://%s.%s", dbAccountName, _env.Environment().CosmosDBDNSSuffixScope)}
```

Comment:
nit: I've seen this same code 3 times in this PR so far, would it be worth extracting into something well named?

Reply:
sure

Reply:
Well, this was the way it was previously, I think lets address this in a separate PR